.github/workflows/release.yml

```diff
@@ -0,0 +1,105 @@
+name: Releases
+
+on:
+  push:
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+"
+
+jobs:
+  build-windows:
+    runs-on: windows-latest
+    timeout-minutes: 60
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: true
+      - uses: msys2/setup-msys2@v2
+        with:
+          update: true
+          path-type: inherit
+          install: >-
+            curl
+            make
+            cmake
+
+      - shell: msys2 {0}
+        run: |
+          pacman -S mingw-w64-x86_64-gcc mingw-w64-x86_64-cmake mingw-w64-x86_64-make mingw-w64-x86_64-curl libcurl openssl-devel libcurl-devel autoconf texinfo mingw-w64-x86_64-libtool --noconfirm
+          make
+
+          mkdir ./txiki-windows-x86
+          cp ./build/tjs.exe ./txiki-windows-x86/tjs.exe
+          cp ./build/libtjs.a ./txiki-windows-x86/libtjs.a
+          cd ./txiki-windows-x86
+          ldd ./tjs.exe | grep '=>' | awk '{print $3}' | xargs -I '{}' cp -v '{}' .
```

Comment:
You need to filter system libraries in c:/windows and the like, we don't want to bundle those.

Reply:
```
 '/c/Windows/SYSTEM32/ntdll.dll' -> './ntdll.dll'
'/c/Windows/System32/KERNEL32.DLL' -> './KERNEL32.DLL'
'/c/Windows/System32/KERNELBASE.dll' -> './KERNELBASE.dll'
'/c/Windows/System32/ADVAPI32.dll' -> './ADVAPI32.dll'
'/c/Windows/System32/msvcrt.dll' -> './msvcrt.dll'
'/c/Windows/System32/sechost.dll' -> './sechost.dll'
'/c/Windows/System32/bcrypt.dll' -> './bcrypt.dll'
'/c/Windows/System32/RPCRT4.dll' -> './RPCRT4.dll'
'/c/Windows/System32/ole32.dll' -> './ole32.dll'
'/c/Windows/System32/msvcp_win.dll' -> './msvcp_win.dll'
'/c/Windows/SYSTEM32/dbghelp.dll' -> './dbghelp.dll'
'/c/Windows/System32/ucrtbase.dll' -> './ucrtbase.dll'
'/c/Windows/System32/GDI32.dll' -> './GDI32.dll'
'/c/Windows/System32/win32u.dll' -> './win32u.dll'
'/c/Windows/System32/gdi32full.dll' -> './gdi32full.dll'
'/c/Windows/SYSTEM32/IPHLPAPI.DLL' -> './IPHLPAPI.DLL'
'/c/Windows/System32/USER32.dll' -> './USER32.dll'
'/c/Windows/System32/combase.dll' -> './combase.dll'
'/c/Windows/System32/SHELL32.dll' -> './SHELL32.dll'
'/c/Windows/System32/WS2_32.dll' -> './WS2_32.dll'
'/mingw64/bin/libwinpthread-1.dll' -> './libwinpthread-1.dll'
'/c/Windows/SYSTEM32/USERENV.dll' -> './USERENV.dll'
'/mingw64/bin/libcurl-4.dll' -> './libcurl-4.dll'
'/mingw64/bin/libffi-8.dll' -> './libffi-8.dll'
'/c/Windows/System32/CRYPT32.dll' -> './CRYPT32.dll'
'/c/Windows/System32/WLDAP32.dll' -> './WLDAP32.dll'
'/c/Windows/SYSTEM32/CRYPTBASE.DLL' -> './CRYPTBASE.DLL'
'/mingw64/bin/libidn2-0.dll' -> './libidn2-0.dll'
'/mingw64/bin/libbrotlidec.dll' -> './libbrotlidec.dll'
'/mingw64/bin/libcrypto-3-x64.dll' -> './libcrypto-3-x64.dll'
'/mingw64/bin/libnghttp2-14.dll' -> './libnghttp2-14.dll'
'/mingw64/bin/libnghttp3-9.dll' -> './libnghttp3-9.dll'
'/mingw64/bin/libssh2-1.dll' -> './libssh2-1.dll'
'/mingw64/bin/libssl-3-x64.dll' -> './libssl-3-x64.dll'
'/mingw64/bin/zlib1.dll' -> './zlib1.dll'
'/mingw64/bin/libzstd.dll' -> './libzstd.dll'
'/mingw64/bin/libintl-8.dll' -> './libintl-8.dll'
'/mingw64/bin/libiconv-2.dll' -> './libiconv-2.dll'
'/mingw64/bin/libunistring-5.dll' -> './libunistring-5.dll'
'/mingw64/bin/libbrotlicommon.dll' -> './libbrotlicommon.dll'
'/mingw64/bin/libpsl-5.dll' -> './libpsl-5.dll'
'/c/Windows/SYSTEM32/dbgcore.DLL' -> './dbgcore.DLL'
```

Basically anything starting with `/mingw` we need to bundle.